src/pages/LogInForm.js

```diff
@@ -11,15 +11,17 @@ import {
   FormControlLabel,
   Checkbox,
 } from "@mui/material";
-
 import LoadingButton from "@mui/lab/LoadingButton";
 import { useForm, Controller } from "react-hook-form";
 import Visibility from "@mui/icons-material/Visibility";
 import VisibilityOff from "@mui/icons-material/VisibilityOff";
+
 import FormProvider from "../components/form/FormProvider";
 import FTextField from "../components/form/FTextField";
 import FCheckBox from "../components/form/FCheckBox";
 
+import { FormProvider, FTe, c } from "./folder";
+
```

Comment:
Should update remove `c`

Reply:
decide to keep the original 3 separate import statement, since the compact syntax {a,b,c} keep producing error. Will resolve this issue once the final product is done. !TODO: will add comment to be reminder